src/FaveListItem/FaveListItem.test.js

```diff
@@ -15,23 +19,17 @@ describe("FaveListItem", () => {
         deckName={deckName}
         deckIndex={index}
         key={deckName}
-        removeFaveListItem={removeFaveListItemMock}
-      />
+        faveDecks={faveDecks}
+        saveArray={saveArrayMock}/>
     )
   });
 
   it('should match the snapshot', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
-  it('should call removeFaveListItem method from props when removeFaveListItem is called', () => {
-    wrapper.find(".fa-trash-alt").simulate("click", {
-      target: { 
-        closest: () => {
-          return { id: 0 }
```

Comment:
Using `props.cardIndex` instead of `event.target` in the removeFaveListItem method simplifies the test.

Reply:
by a lot! nice work!